packages/core/strapi/src/cli/commands/export/action.ts

```diff
@@ -3,6 +3,12 @@ import fs from 'fs-extra';
 import chalk from 'chalk';
 import type { LoadedStrapi } from '@strapi/types';
 
+import {
+  engine as engineDataTransfer,
+  strapi as strapiDataTransfer,
```

Comment:
I just noticed the data-transfer exports are all a bit weird. It would be safer if we rename the export from `strapi` to anything else. But we can do that in a later refactor.

Reply:
Yeah that's why I alias everything :D I'm wondering if it would be worth having only a single entrypoint `dataTransfer.xxx` too 🤔 Let's do that later ;)